### ISSUE TITLE
[FF33 TU1.5.28.1] Just can't open "about:config" page,disable TU will be fine.

### ISSUE THREAD

**Issue by zippo1933** [ithinc/tabutils#121]
[FF33 TU1.5.28.1] Just can't open "about:config" page,disable TU will be fine.
Also with "about:newtab","about:plugins" and so on.

**Comment by atomGit** [comment]
i can confirm this on F 33, win 7, TU 1.5.28.1 - none of the 'about:' links work until TU is disabled and FF restarted

**Comment by spvkgn** [comment]
Same here. I can't open any URL typed in address bar with FF33 and TU1.5.28.1 on Linux. With disabled TU works fine.

**Comment by yfdyh000** [comment]
#108 fix it. you can download it from https://github.com/yfdyh000/tabutils/releases.

**Comment by GridTrekkor** [comment]
The fix doesn't work.  Entering a URL in the address bar in a new tab does nothing.

**Comment by jiggie42** [comment]
I had the same problem as @spvkgn , regardless of the address, it won't load or do anything with FF33 TU1.5.28.1 on Windows 7.  #108 fixed the problem for me.

**Comment by xyjprc** [comment]
Same here. Cannot open any URL in address bar using  Tab Utilities 1.5.28.1 and FF33 on Windows 8 x64 and Mac Mavericks / Yosemite. Disabling TU will solve the problem.

**Comment by sumyungguy** [comment]
had the problem of any url in address bar not working, yfdyh000's fix works for me on Mac, thanks!

**Comment by GridTrekkor** [comment]
I tried the fix again (Win 8.1, FF 33) and it still doesn't work :(

**Comment by phts** [comment]
Any errors in the browser console (Ctrl+Shift+J)?

**Comment by GridTrekkor** [comment]
Nope, no errors.  I even tried loading the updated code in tabutils.js into the xpi and installing, but no go.

**Comment by thefreecat** [comment]
I have the same error on Ubuntu 14.04/FF33 : no url in address bar working for new tabs/windows.
The console says : params is not defined (in tulib.js line 53)
Hope this helps.

**Comment by GridTrekkor** [comment]
@thefreecat Does the fix work for you?

**Comment by thefreecat** [comment]
Oops sorry, didn't understand how to apply the fix or if it was the same bug as I had.
I tried the fix and yes, it works perfectly. Thanks !

**Comment by GridTrekkor** [comment]
How do you apply the fix other than just installing it the way you would any other extension?

**Comment by thefreecat** [comment]
I don't quite get your question. I installed it by simply clicking on the .xpi
It says version 1.5.28.1 in the extensions list

**Comment by GridTrekkor** [comment]
That's what I did, but it still not working for me.  I disabled Classic Theme Restorer, but that didn't help.

**Comment by RedHorizon** [comment]
Reproduce steps:
Launch FF33, open new tab, type any url, about:config or word in url bar - it won't work

**Comment by imraro** [comment]
> #108 fix it. you can download it from https://github.com/yfdyh000/tabutils/releases.

Fix works only when NoScript is disabled.

**Comment by GridTrekkor** [comment]
OK, I disabled NoScript and now it works for me.  Unfortunately, I need NoScript to be working :(  Is there some exception we can add to NoScript to make this work?

**Comment by MalboneStreet** [comment]
After some trial and error I found that, for me, setting extensions.tabutils.openurlintab=true fixed the urlbar problem.  That said, there appears to be no way to force typed-in addresses to open in the same tab. (Using FF 33.0.2 and TU 1.5.28.1; Win7 & MacOS_10.8.5)

**Comment by antitoxic** [comment]
Same problem here. WIndows with latest firefox. 
Neither entering urls work nor searching in address bar.

Would love to use the extension but that's a showstopper. If there's a priority system that's a critical.

**Comment by MFWills** [comment]
FF 34, TU1.5.28.1. Address entry a no go. Nothing happens until I disable TU.

**Comment by yfdyh000** [comment]
@imraro @GridTrekkor
Try the https://github.com/yfdyh000/tabutils/releases 2015.02.16.

**Comment by imraro** [comment]
Works like charm! Thank you!
